### DISCUSSION TITLE
How to do Speaker Adaptation?

### DISCUSSION THREAD

**Discussion by sqrk** [espnet/espnet#4565] [Q&A]
Hello,

I'm new to ASR and I've read that a lot of papers for my task (impaired speech recognition) use speaker adaptation (MAP, MLLR...) and speaker adaptive training and I'm not sure where to start. Is there a recipe for it or a link that could help me figure out how to implement it?

Thank you!

**Comment by sw005320** [comment]
MAP, MLLR, and so on. are based on the Gaussian mixture model, not the neural network.
I recommend you check Kaldi for it (https://github.com/kaldi-asr/kaldi)
You can use fine-tuning or transfer learning techniques if you're thinking of using the neural network.
You can check some materials, e.g., https://espnet.github.io/espnet/notebook/espnet2_asr_transfer_learning_demo.html
